.dscanner.ini

```diff
@@ -63,6 +63,6 @@ auto_ref_assignment_check="disabled" ; FIXME
 ; Checks for incorrect infinite range definitions
 incorrect_infinite_range_check="enabled"
 ; Checks for asserts that are always true
-useless_assert_check="disabled"; FIXME
+useless_assert_check="skip-unittest"
```

Comment:
"skip-unittest" makes sense for this check because asserts are often used to check that something compiles.